src/openforms/js/components/admin/form_design/registrations/ms_graph/MSGraphOptionsForm.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import OptionsConfiguration from 'components/admin/form_design/registrations/shared/OptionsConfiguration';
+import Fieldset from 'components/admin/forms/Fieldset';
+import {
+  ValidationErrorContext,
+  ValidationErrorsProvider,
+  filterErrors,
+} from 'components/admin/forms/ValidationErrors';
+
+import {DriveID, FolderPath} from './fields';
+
+const MSGraphOptionsForm = ({name, label, formData, onChange}) => {
+  const validationErrors = useContext(ValidationErrorContext);
+  const relevantErrors = filterErrors(name, validationErrors);
+  return (
+    <OptionsConfiguration
+      name={name}
+      label={label}
+      numErrors={relevantErrors.length}
+      modalTitle={
+        <FormattedMessage
+          description="Demo registration options modal title"
+          defaultMessage="Plugin configuration: demo"
```

Comment:
I think the `modalTitle` still needs to change? Maybe something like:
```suggestion
          description="MS graph registration options modal title"
          defaultMessage="Plugin configuration: MS graph"
```